content/docs/user-guide/networking/multiplayer/configuration.md

```diff
@@ -31,7 +35,7 @@ Make the following edits to the `<ProjectName>.Static` target:
                 ...
                 <projectname>_autogen_files.cmake
     ```
-    {{< note >}}The `<projectname>_autogen_files.cmake` file is created later in the [Adding AutoGen CMake file](#adding_autogen_file) section. For now, create a reference to it in the `CMakeList.txt`{{< /note >}}
+    {{< note >}}You create the `<projectname>_autogen_files.cmake` file later in the [Add the AutoGen CMake file](#add-the-autogen-cmake-file) section. For now, create a reference to it in the `CMakeList.txt` file.{{< /note >}}
```

Comment:
```suggestion
```

Can safely remove if this is covered later in the process - a note would mostly be for advice or covering errata/minor details that somebody might care about.